CavalcanteLucas/todolist
# Refactor/OL74-auth_reducers

**Link to task:** 
https://labcodes.atlassian.net/browse/OL-74

**How to test:**
- Check if failure message is appearing at login form when providing invalid credentials;
- Check if there is no confirmation e-mail sent on user registration;

**Description of your solution:**
This PR refactors the variable names from `mapStateToProps` and `mapDispatchToProps` over the `auth` front-end to keep the pattern introduced with the set of components for `todo_list`. The actions and reducers files have also been reorganized to ease the reading. This PR also fixes errors in the front and backend of the `auth` app.
